test/NosCore.Tests/HandlerTests/InventoryPacketControllerTests.cs

```diff
@@ -805,7 +805,7 @@ public void Test_RemoveSp()
         [TestMethod]
         public void Test_Transform()
         {
-            _session.Character.WorldConfiguration = new WorldConfiguration { MaxSpPoints = 10_000, MaxAdditionalSpPoints = 1_000_000 };
+            _session.WorldConfiguration = new WorldConfiguration { MaxSpPoints = 10_000, MaxAdditionalSpPoints = 1_000_000 };
```

Comment:
should be line 109 not here var conf = new WorldConfiguration { BackpackSize = 2, MaxItemAmount = 999 };